pyvrp/stop/ReachedBKS.py

```diff
@@ -0,0 +1,36 @@
+class ReachedBKS:
+    """Indicates that the best known solution has been reached.
+
+    This stopping criterion can be used to terminate the optimization
+    process once a known best solution has been found. This is useful
+    in scenarios where the optimal solution is known beforehand, allowing
+    the algorithm to stop early and save computational resources.
+
+    Parameters
+    ----------
+    bks_value
+        The objective value of the best known solution.
+    """
+
+    def __init__(self, bks_value: int):
+        """Initializes the ReachedBKS stopping criterion.
+
+        Parameters
+        ----------
+        bks_value
+            The objective value of the best known solution. Must be a
+            positive integer (PyVRP uses integer costs internally).
+        """
+        if bks_value <= 0:
```

Comment:
Why is zero not allowed? For instances that are largely about finding a feasible solution, a zero objective is not that uncommon.